src/components/mention-area.component.js

```diff
@@ -1,36 +1,30 @@
 import React, { Component } from 'react';
 import fuzzysort from 'fuzzysort';
-import {
-  Text,
-  View,
-  TouchableOpacity,
-  Animated,
-  ScrollView,
-  StyleSheet,
-} from 'react-native';
+import styled from 'styled-components/native';
+import { TouchableOpacity, Animated, ScrollView } from 'react-native';
 
 import { animations, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  suggestionsRowContainer: {
-    padding: 5,
-    flexDirection: 'row',
-    paddingRight: 15,
-    paddingBottom: 15,
-  },
-  userDetailsBox: {
-    flex: 1,
-    margin: 5,
-  },
-  displayNameText: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimary,
-  },
-  realNameText: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimarySemiBold,
-  },
-});
+const SuggestionsRowContainer = styled.View`
+  flex-direction: row;
+  padding-top: 5;
+  padding-left: 5;
+  padding-right: 15;
+  padding-bottom: 15;
+`;
+
+const UserDetailsBox = styled.View`
+  flex: 1;
+  margin-left: 5;
+  margin-right: 5;
+  margin-top: 5;
+  margin-bottom: 5;
+`;
+
+const DisplayNameText = styled.Text`
+  fontSize: ${normalize(12)};
+  ${{ ...fonts.fontPrimary }}'
```

Comment:
@binkpitch Can we simply write `${fonts.fontPrimary}` here? Please correct me if I am wrong. 😅

Reply:
@chinesedfan  I think ```${{ ...fonts.fontPrimary }}``` can be change to  ```${fonts.fontPrimary}```. I'll make a new commit to fix that.